generators/entity/templates/src/main/resources/config/liquibase/changelog/_added_entity_constraints.xml

```diff
@@ -12,6 +12,14 @@
             relationshipName = relationships[idx].relationshipName,
             ownerSide = relationships[idx].ownerSide,
             otherEntityName = relationships[idx].otherEntityName;
+            if (otherEntityName === 'user') {
+                otherEntityTableName = jhiPrefix + '_user';
```

Comment:
I dont think we should prefix `jhiPrefix ` for user table as the table name for user will always be `jhi_user` see [this](https://github.com/jhipster/generator-jhipster/blob/master/generators/server/templates/src/main/resources/config/liquibase/changelog/_initial_schema.xml#L23)
so if user sets a jhiPrefix then this will fail, it works only with default prefix which is `jhi`